packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -364,6 +370,12 @@ class DragTarget<T> extends StatefulWidget {
   /// into this drag target.
   final DragTargetBuilder<T> builder;
 
+  /// Called when the drag position changed. Receives the latest drag offset,
+  /// which is the pointer position if [DragAnchor.pointer] has been used
+  /// and the position of the dragged widget if [DragAnchor.child] has been
+  /// used.
```

Comment:
I think the distinction is clear now from the class comment. I'd rather not duplicate too much of the description. Wdyt?